databuilder/setup.cfg

```diff
@@ -13,7 +13,7 @@ exclude =
   databuilder/sql_parser/usage/presto/antlr_generated
 max-complexity = 10
 max-line-length = 120
-ignore = NONE
+ignore = E117,E252,E741,F541,F841,W504,W605
```

Comment:
could you give some context what we ignore here?

Reply:
I think that with bumping flake8 it just became more strict with checks. `make lint` on `main` branch passes while here (after bumping from 3.5.0 to 3.9.2) it yields

```
(.venv) root@mariusz-XPS-13-7390-2-in-1:/amundsen/databuilder# make lint
flake8 .
./databuilder/task/task.py:72: [F541] f-string is missing placeholders
./databuilder/publisher/elasticsearch_publisher.py:69: [W504] line break after binary operator
./databuilder/publisher/elasticsearch_publisher.py:80: [E741] ambiguous variable name 'l'
./databuilder/publisher/neo4j_csv_publisher.py:460: [F541] f-string is missing placeholders
./databuilder/publisher/atlas_csv_publisher.py:89: [F541] f-string is missing placeholders
./databuilder/publisher/atlas_csv_publisher.py:104: [F541] f-string is missing placeholders
./databuilder/publisher/atlas_csv_publisher.py:217: [F541] f-string is missing placeholders
./databuilder/models/atlas_serializable.py:52: [F541] f-string is missing placeholders
./databuilder/models/schema/schema.py:29: [E252] missing whitespace around parameter equals
./databuilder/models/schema/schema.py:29: [E252] missing whitespace around parameter equals
./databuilder/models/schema/schema.py:30: [E252] missing whitespace around parameter equals
./databuilder/models/schema/schema.py:30: [E252] missing whitespace around parameter equals
./databuilder/extractor/dbt_extractor.py:124: [E117] over-indented
./databuilder/extractor/hive_table_last_updated_extractor.py:228: [F541] f-string is missing placeholders
./databuilder/extractor/delta_lake_metadata_extractor.py:42: [W504] line break after binary operator
./databuilder/extractor/delta_lake_metadata_extractor.py:43: [W504] line break after binary operator
./databuilder/extractor/delta_lake_metadata_extractor.py:44: [W504] line break after binary operator
./databuilder/extractor/delta_lake_metadata_extractor.py:45: [W504] line break after binary operator
./databuilder/extractor/delta_lake_metadata_extractor.py:46: [W504] line break after binary operator
./databuilder/extractor/atlas_search_data_extractor.py:237: [F841] local variable 'e' is assigned to but never used
./databuilder/extractor/atlas_search_data_extractor.py:354: [F541] f-string is missing placeholders
./databuilder/extractor/dashboard/apache_superset/apache_superset_chart_extractor.py:57: [F541] f-string is missing placeholders
./databuilder/extractor/dashboard/apache_superset/apache_superset_metadata_extractor.py:59: [F541] f-string is missing placeholders
./databuilder/extractor/dashboard/tableau/tableau_dashboard_last_modified_extractor.py:43: [W504] line break after binary operator
./databuilder/extractor/dashboard/tableau/tableau_dashboard_extractor.py:49: [W504] line break after binary operator
./example/scripts/sample_dbt_loader.py:68: [F541] f-string is missing placeholders
./example/scripts/sample_deltalake_metadata.py:52: [F541] f-string is missing placeholders
./example/scripts/sample_pandas_profiling_column_stats_loader.py:16: [F541] f-string is missing placeholders
./example/scripts/sample_feast_loader.py:54: [F541] f-string is missing placeholders
./tests/unit/publisher/test_mysql_csv_publisher.py:22: [F541] f-string is missing placeholders
./tests/unit/publisher/test_neo4j_csv_publisher.py:23: [F541] f-string is missing placeholders
./tests/unit/extractor/test_csv_extractor.py:24: [F541] f-string is missing placeholders
./tests/unit/extractor/test_kafka_source_extractor.py:18: [F541] f-string is missing placeholders

```

Reply:
could you file a ticket and see if we could get some community member to help fixing these?

Reply:
sure, will do after merging. sounds like a nice idea for `good-first-issue`